### ISSUE TITLE
Refresh should not cause the map to reset to your position

### ISSUE THREAD

**Issue by jancborchardt** [owncloud/maps#86]
Rather than always resetting to your position, the Maps app should save the bounding box you last saw and show that again.

When you want to change to your position, there’s already a button for it.

**Comment by DJaeger** [comment]
:+1:

**Comment by jancborchardt** [comment]
@DJaeger you want to pick that up? :)

**Comment by DJaeger** [comment]
That was not my intention for the comment, but sure I can look into it.

**Comment by jancborchardt** [comment]
@Djaeger yeah, everyone can just feel free to work on whatever they want of course. :) This seemed to be a nice thing we can fix for a smoother experience.

**Comment by v1r0x** [comment]
The current position on the map is stored in the URL as lat and lng parameter. Thus the position should be set to this position after a refresh. Do we need more?

**Comment by jancborchardt** [comment]
In the past, the geolocation automatically switched the position to there. If that’s fixed too we’re good to go. :)

**Comment by v1r0x** [comment]
Riiight....Totally forgot about that. Think we still have to fix this.

ps: could you have a look at the pending PRs. We're almost done for 0.1 ;)
